nimble/host/mesh/src/glue.c

```diff
@@ -342,13 +342,13 @@ net_buf_reserve(struct os_mbuf *om, size_t reserve)
 void
 k_work_init(struct ble_npl_callout *work, ble_npl_event_fn handler)
 {
-    ble_npl_callout_init(work, ble_npl_eventq_dflt_get(), handler, NULL);
+    ble_npl_callout_init(work, nimble_port_get_dflt_eventq(), handler, NULL);
```

Comment:
does this build on Mynewt? this function is not defined when building for mynewt since it does not work on "ported" default eventq

Reply:
did not try. I'm building for freeRTOS

Reply:
All I know that without this change it's not building on freeRTOS

Reply:
and with this change it does not build on Mynewt